pypot/server/httpserver.py

```diff
@@ -218,12 +219,23 @@ def get_motors_register_value(register_name):
 
             return registers_motors
 
-
     def run(self, quiet=None, server='tornado'):
         """ Start the bottle server, run forever. """
         if quiet is None:
             quiet = self.quiet
-        bottle.run(self.app,
-                   host=self.host, port=self.port,
-                   quiet=quiet,
-                   server=server)
+        try:
+            bottle.run(self.app,
+                       host=self.host, port=self.port,
+                       quiet=quiet,
+                       server=server)    
+        except RuntimeError as e:
```

Comment:
The IOLoop is defined [here](http://www.tornadoweb.org/en/stable/_modules/tornado/ioloop.html). I didn"t investigate a lot but I feel that only one IOLoop is required to run as many as we want tornado instances